bin/compile

```diff
@@ -45,3 +40,41 @@ cp "${basedir}/../conf/httpd.conf.erb" "${APACHE_DIR}/conf/httpd.conf.erb"
 
 mkdir -p "${BUILD_DIR}/bin"
 cp "${basedir}/startup.sh" "${BUILD_DIR}/bin/startup.sh"
+
+if grep -q "auth-mellon" "${BUILD_DIR}/.apache-mods" ; then
```

Comment:
This should be done at runtime and not during the build.

So if the user fails to setup their key correctly, they can change the values and just do a `restart` instead of having to trigger a deployment without any code change done to their repository